torch/quantization/quantization_mappings.py

```diff
@@ -86,6 +86,9 @@
 FLOAT_TO_QUANTIZED_OPERATOR_MAPPINGS = {
     F.elu: torch._ops.ops.quantized.elu,
     F.leaky_relu: torch._ops.ops.quantized.leaky_relu,
+    F.sigmoid: torch._ops.ops.quantized.sigmoid,
```

Comment:
Why is this specific to sigmoid? Do we need three forms for every functional?

Reply:
not necessarily, `x.leaky_relu()` is not supported I think: https://pytorch.org/docs/stable/search.html?q=leaky_relu&check_keywords=yes&area=default#